src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.Analyzer.cs

```diff
@@ -24,12 +24,6 @@ protected override bool TreatOutAsRef
             protected override bool IsInPrimaryConstructorBaseType()
                 => this.SelectionResult.GetContainingScopeOf<PrimaryConstructorBaseTypeSyntax>() != null;
 
-            protected override VariableInfo CreateFromSymbol(
-                ISymbol symbol, ITypeSymbol type, VariableStyle style, bool variableDeclared)
-            {
-                return CreateFromSymbolCommon(symbol, type, style);
-            }
```

Comment:
this was a virtual to allow VB to specialize behavior.  but that specialization could be pulled up to the caller.